Refactor imports to be consistent with `edition = "2023_10"` (#473)
cairo-book/cairo-book: src/ch02-02-data-types.md

### Type casting

> In Cairo, you can convert scalar types from one type to another by using the `try_into` and `into` methods provided by the `TryInto` and `Into` traits, respectively. These traits are included in the prelude and don't need to be explicitly imported.

Reviewer: ```suggestion
In Cairo, you can convert scalar types from one type to another by using the `try_into` and `into` methods provided by the `TryInto` and `Into` traits from the core library.
```

imo: no need to specify unless it requires a specific import
